data/builder/builder.go

```diff
@@ -0,0 +1,116 @@
+package builder
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/ipfs/go-unixfsnode/data"
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/fluent/qp"
+)
+
+func BuildUnixFs(fn func(*Builder)) (data.UnixFSData, error) {
```

Comment:
add a comment to this toplevel method with pointer to example of using it to build a node